### PR TITLE
adding some events - removing woffols

### PR DESCRIPTION
**Pull request by my0sin**
will need to work on info but these are the events we have scheduled.

### CODE REVIEW

File: content/events/mpls-activities-fair.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
++++
+title = "Mpls Activities Fair"
+date = 2018-09-12
+
+template = "post.html"
++++
+
+Come see us at the Mpls Activities Fair!
```
Comment (remexre) [inline content/events/mpls-activities-fair.md]: Should the abbreviation be expanded here?

File: content/events/welcome-week.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
++++
+title = "Welcome Week CSE Student Org. Fair"
+date = 2018-08-30
+
+template = "post.html"
++++
+
+12:30 - 4:30 PM @ Lind Hall Couryard
```
Comment (remexre) [inline content/events/welcome-week.md]: Link to official page?

File: content/events/explore-u.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
++++
+title = "Explore U"
+date = 2018-09-01
+
+template = "post.html"
++++
+
+Come see us at Explore U! 
+1:30 - 4:30 PM @ Mariucci Arena
```
Comment (remexre) [inline content/events/explore-u.md:9]: Link to official page?

**Review by remexre — COMMENTED** [review]
Do we have longer versions of any of these?

Also @iptq do we have a solution for needing e.g. `mh-2.md`?

**Comment by mzhang28** [comment]
should we clarify between week 1/2/3 of sdc?

**Comment by mzhang28** [comment]
also even though minnehack is two days it should only appear on the events calendar as one

**Comment by my0sin** [comment]
Does it allow for a span of dates?
 Re mh mh2

**Comment by mzhang28** [comment]
no(t yet) but it shouldn't really matter? it's not each day needs to be separately documented

**Comment by mzhang28** [comment]
unless u have a need for an API that counts the number of days of the year in which we have events

**Comment by remexre** [comment]
and in that case, you probably want an hours, so you should use an `[extra]`

**Comment by my0sin** [comment]
I think for sdc we can just do sdc - design week start, sdc - implementation week start, sdc - judging

**Comment by mzhang28** [comment]
honestly i think we should just have each event show up once on the timeline, and then in the event page (wip) have some kind of detailed description

**Pull request merged**